supported/osg-htc/osg-hosted-ce/templates/configmap.yaml

```diff
@@ -71,8 +71,8 @@ data:
     TCP_FORWARDING_HOST = {{ .Values.Networking.RequestIP }}
 {{ end }}
 
-    SLATE_HOSTED_CE = True
-    SCHEDD_ATTRS = $(SCHEDD_ATTRS) SLATE_HOSTED_CE
+    HOSTED_CE = True
+    SCHEDD_ATTRS = $(SCHEDD_ATTRS) HOSTED_CE
```

Comment:
@brianhlin can you look at this part? Not sure if this is kosher, not really sure what that's used for or if it's expecting something called SLATE_HOSTED_CE.

Reply:
This shows up in the SchedD ClassAd of the CE:

```
$ condor_ce_status -pool collector.opensciencegrid.org:9619 -schedd -const 'SLATE_HOSTED_CE == True' -af name | head
aei-hypatia-ce1.svc.opensciencegrid.org
amnh-ares-ce1.svc.opensciencegrid.org
amnh-hel-ce1.svc.opensciencegrid.org
amnh-mendel-ce1.svc.opensciencegrid.org
asu-sol-ce1.svc.opensciencegrid.org
canary2.osg.chtc.io
cardiff-igwn-ce1.svc.opensciencegrid.org
chtc-spark-ce1.svc.opensciencegrid.org
clarkson-acres-ce2.svc.opensciencegrid.org
clemson-palmetto-ce1.svc.opensciencegrid.org
```

I'm not sure if anything uses this for queries so it's probably safer to add/append `OSG_HOSTED_CE` and eventually transition to it instead of removing `SLATE_HOSTED_CE`

Reply:
Fixed in 305f7f36661e340717be0683c2e97b7ce481bd0e